app/react/Settings/components/EntityTypesList.js

```diff
@@ -72,14 +73,21 @@ export class EntityTypesList extends Component {
     );
   }
 
+  // eslint-disable-next-line class-methods-use-this
+  sortTemplates(templates) {
+    const def = templates.filter(t => t.default);
+    const sorted = templates.filter(t => !t.default).sort((a, b) => a.name > b.name);
```

Comment:
I think we agreed that the default template wasn't going to be treated specially, but just sorting it as well, didn't we?

Reply:
I might not have catched that, changed